HTTPServerProject.Tests/HTTPServerProxyTests/CheckPathTests.cs

```diff
@@ -0,0 +1,16 @@
+using HTTPServerProxy.Check.Paths;
+
+namespace HTTPServerProxyTests.Check.Paths;
+
+public class HttpPathTests
+{
+    [Theory]
+    [InlineData("todo", true)]
```

Comment:
`InlineData` is cool! Nice job reducing the number of tests you would have to write.

Reply:
Thank you!